src/data/systemIntake.ts

```diff
@@ -267,7 +267,7 @@ export const prepareSystemIntakeForApp = (
   };
 };
 
-export const convertIntakeToCSV = (intake: SystemIntakeForm) => {
+export const convertIntakeToCSV = (intake: any) => {
```

Comment:
Is there a solution that doesn't involve losing the type safety on this function?

Reply:
@ClayBenson94 This is a little tricky - the code that already existed was doing a lot that breaks type safety (for example, overwriting the request property, which is supposed to be an object, with a string). I don't know what the preferred approach would be; maybe to add the new property to SystemIntakeForm). In general, a lot of the related code here is playing fast and loose with the typing.

I should also mention, I had an un-pushed commit when I made this PR. Just pushed it - gets rid of the comments and tweaked a couple things.

Reply:
Not sure, but maybe `(intake: SystemIntakeForm & { requesterNameAndComponent: string })` is good enough to describe what's happening in this change while also encouraging typing for the next dev. Maybe it's ineffective but just in a different way though. It's just too bad to have to fall back to any in these annoying situations.

I'm guessing SystemIntakeForm should just be kept for the form related concerns.

Reply:
@adamodd Ah, that makes sense. I didn't know you could do something like that in TypeScript. I'll add that.

Reply:
Okay, this change is pushed in my latest commit.